src/lib/chat/matrix-client.ts

```diff
@@ -313,6 +314,28 @@ export class MatrixClient implements IChatClient {
     return await this.getMessageByRoomId(roomId, editResult.event_id);
   }
 
+  async markRoomAsRead(roomId: string): Promise<void> {
+    const room = this.matrix.getRoom(roomId);
+
+    if (!room) {
+      return;
+    }
+
+    const events = room.getLiveTimeline().getEvents();
+    const latestEvent = events[events.length - 1];
```

Comment:
just wondering if we need to load all the events? Is there a method the fetch the latest event directly?

Reply:
Unfortunately not that I have found. If you come across one let us know 💪